File: t/app/src/main/java/com/geektech/t/MainActivity.java
package com.geektech.t;

import androidx.appcompat.app.AppCompatActivity;

import android.annotation.SuppressLint;
import android.graphics.drawable.Drawable;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.ImageView;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity {
    TextView counter;
    TextView s;
    Integer value;
    Button plus;
    ImageView imageView;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        counter = findViewById(R.id.counter);
        plus = findViewById(R.id.plus);
        imageView = findViewById(R.id.image);

        plus.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                s = findViewById(R.id.s);
                s.setText("FFFFF");

                Drawable image= R.drawable.ic_image;
                imageView.setImageDrawable(image);

                value = Integer.parseInt(counter.getText().toString()) + 1;
                counter.setText(value.toString());
            }
        });



    }
}

File: Life/app/src/main/java/com/geektech/life/MainActivity.kt
package com.geektech.life

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import com.geektech.life.databinding.ActivityMainBinding

class MainActivity : AppCompatActivity() {
    private lateinit var binding: ActivityMainBinding
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = ActivityMainBinding.inflate(layoutInflater)
        setContentView(binding.root)

        binding.swFriends.setOnClickListener {
            binding.swFriends.isChecked = binding.swFriends.isChecked
            if(binding.swFriends.isChecked && binding.swMoney.isChecked){
                binding.swMoney.isChecked = false
            }
        }

        binding.swLove.setOnClickListener {
            binding.swLove.isChecked = binding.swLove.isChecked
            if(binding.swFriends.isChecked && binding.swMoney.isChecked){
                binding.swFriends.isChecked = false
            }
        }

        binding.swMoney.setOnClickListener {
            binding.swMoney.isChecked = binding.swMoney.isChecked
            if(binding.swFriends.isChecked && binding.swLove.isChecked){
                binding.swLove.isChecked = false
            }
        }

    }
}